packages/website/src/components/FinancialContributors.tsx

```diff
@@ -45,16 +70,24 @@ export function FinancialContributors(): JSX.Element {
       </p>
       <div className={styles.sponsorsContainer}>
         <Sponsors
-          description="Contributors of a monthly amount of $100 or more."
+          className={styles.tierSponsorArea}
+          description="Donors of $750 and/or a monthly amount of $100 or more."
           showName
           tier="sponsor"
           title="Sponsors"
         />
         <Sponsors
-          description="Contributors of a monthly amount of $3 or more."
+          className={styles.tierSupporterArea}
+          description="Donors of $150 and/or a monthly amount of $3 or more."
```

Comment:
@JamesHenry what do you think about raising this in OpenCollective to, say, $10/month? The $3/month number is low enough that we're pretty attractive bait to spam at the moment, and I'd like donors who sign up for the monthly tier to get immediate placement.

Reply:
I like the increase when it comes to determining what gets shown/publicity, but I've looked at it in OC and I think it would be a real shame to prevent individuals being able to give a small amount each month if we force the minimum up to 10 in the system.

I think it might be the best of both worlds to simply add the disclaimer that only sponsors who pledge 10+ per month will be shown here, but that smaller donations are still gratefully accepted.

I think of it this way psychologically -  $3 is in the ballpark of buying one of us a cup of coffee each month to thank us, $10 is in the ballpark of their Netflix/Spotify etc subscription

Reply:
And once we make the final decision, I can update this wording of the bot comment accordingly

![image](https://user-images.githubusercontent.com/900523/160594727-0b1a96a6-dca9-465f-b785-811394c189a7.png)

Reply:
That makes sense. The tricky thing is it's a bit more work to iterate through recent donations and figure out what the person's _current_ sponsor tier is. This especially since people can set up custom donations if they want.

How about the best of both worlds: keeping the $3/mo tier, adding a new $10/mo tier, and keeping the $100/mo tier?

I should probably cc @bradzacher as an fyi too.

Reply:
I like the idea of a new tier. Sort of a "supporter plus" to help weed out some of the spam.
We could also just remove the URLs from the page entirely and just show logos.
That's sort of how it used to work before the website - we'd just have the single autogenerated, static image from opencollective which showed all the logos.

Reply:
I think that makes a lot of sense for the lower tiers, yeah. Will make that change now.

@JamesHenry I don't have admin controls on the OpenCollective, so I'll wait for you to make a new tier.

Reply:
Added the Supporter Plus tier with a $10 monthly minimum. I haven't listed anything about URLs in the description on OC, and I'm not 100% clear on what's being suggested by @bradzacher.

Are you advocating for just the Supporter tier no longer getting URLs? Or Supporter Plus as well?

I would think the URL linking could be a good incentive for folks to graduate from Supporter to Supporter Plus? They could have technically already gifted $10 / month on Supporter because it doesn't force you into a specific amount (just a specific minimum) so I would think there needs to be some incentive there.

Certainly for Sponsor tier we will want to link out to them right?

Reply:
> good incentive for folks to graduate from Supporter to Supporter Plus? 

That makes sense to me! Added links back to those.

> Sponsor tier we will want to link out to them

100%